templates/forms/DropdownField.ss

```diff
@@ -1,5 +1,5 @@
 <select $AttributesHTML>
 <% loop $Options %>
-	<option value="$Value.XML"<% if $Selected %> selected="selected"<% end_if %><% if $Disabled %> disabled="disabled"<% end_if %>><% if Title %>$Title.XML<% else %>&nbsp;<% end_if %></option>
```

Comment:
it fails for me locally. :dancer: 

![image](https://cloud.githubusercontent.com/assets/936064/8296257/a6e892d6-19a2-11e5-817f-e40cabc320ae.png)